### PR TITLE
Create Login and Sign up screens

### PR DESCRIPTION
**Pull request by shayywun**
Issue: https://github.com/ChemoTrackerApp/react-mobile/issues/2

Created the layout for login and signup screens. This only reflects the frontend side. Issue #3 reflects the addition of more logic.
img_4946
img_4947

### CODE REVIEW

File: ChemoTracker/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  textFieldContainer: {
+    height: 60,
+    width: 300,
+    backgroundColor: "#FF6666",
```
Comment (jpatricia) [inline ChemoTracker/SignUp.js]: the background is too dark, I think it should be a lighter color, so the text is more readable.. Maybe you can try opacity? if not then just use lighter color

File: ChemoTracker/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    width:150,
+    marginTop: 20,
+    marginBottom: 20
+  },
+  logo: {
+    height: 250,
+    width: 250
+  },
+  text:{
+    color: "#6374f4",
```
Comment (jpatricia) [inline ChemoTracker/SignUp.js]: either the text color that should change to be more redable, or the background should be lighter

File: ChemoTracker/Login.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  textFieldContainer: {
+    height: 60,
+    width: 300,
+    backgroundColor: "#FF6666",
```
Comment (jpatricia) [inline ChemoTracker/Login.js]: same as signup, I think the color should be lighter

File: ChemoTracker/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+          accessibilityLabel="Tap on Me"
+         />
+       </View>
+       <Text>Already Registered?  <Text style={styles.text}>LOGIN</Text></Text>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```
Comment (jpatricia) [inline ChemoTracker/SignUp.js]: let's all agree to put styling in a separate file?

File: ChemoTracker/Login.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+       />
+       <View style={styles.buttonContainer}>
+         <Button
+          onPress={()=> {}}
+          title="Login"
+          color="#fff"
+          accessibilityLabel="Tap on Me"
+         />
+       </View>
+       <Text>New? <Text style={{color: "#6374f4", fontWeight: 'bold'}}>SIGN UP</Text></Text>
```
Comment (jpatricia) [inline ChemoTracker/Login.js]: I think all styling should be in the `Stylesheet`, and let's put it in a separate file

**Review by jpatricia — CHANGES_REQUESTED** [review]

File: ChemoTracker/SignUp.js (hunk 1 above)
Comment (jesshanta) [reply ChemoTracker/SignUp.js]: I agree. The red is very harsh.

**Review by jpatricia — APPROVED** [review]

**Pull request merged**